RAID/refresh_secondary

```diff
@@ -233,14 +241,20 @@ fi
 rm -f $lockf
 
 # Finally report on recovery progress until it completes
-while ! grep -q recovery /proc/mdstat; do 
-	echo "Waiting for recovery ... "
-	sleep 1;
-done
+if [ "$PROGRESS_METER" -ne 0 ]; then
+	echo "(This may safely be interrupted with 'Control-C' if you don't want to wait)"
+	while ! grep -q recovery /proc/mdstat; do
+		echo "Waiting for recovery ... "
+		sleep 1;
+	done
 #
-while grep -q recovery /proc/mdstat; do 
-	recovery=$(grep recovery /proc/mdstat | cut -c32-)
-	echo -e -n "\r$recovery"
-	sleep 1;
-done
-echo -e "\ndone."
+	while grep -q recovery /proc/mdstat; do
+		recovery=$(grep recovery /proc/mdstat | cut -c32-)
+		echo -e -n "\r$recovery"
+		sleep 1;
+	done
+	echo -e "\ndone."
+else
+	echo "You check on the recovery progress with 'mdstat'."
```

Comment:
And again.

Reply:
Unfortunately: consistency is the hobgoblin of little minds